sdk/elixir/lib/dagger/core/engine_conn.ex

```diff
@@ -95,5 +93,7 @@ defmodule Dagger.Core.EngineConn do
     Dagger.Session.stop(pid)
   end
 
-  def engine_version(), do: @dagger_cli_version
+  def engine_version() do
+    Dagger.Core.Version.engine_version()
+  end
```

Comment:
I just notice that no one use this function. I can help removing this function after this PR get merged.